raft/progress.go

```diff
@@ -64,12 +64,17 @@ type Progress struct {
 	RecentActive bool
 
 	// inflights is a sliding window for the inflight messages.
+	// Each inflight message might contain more than one entries
+	// and has a size limit defined in raft config as MaxSizePerMsg.
```

Comment:
I am considering to just make this a direct bytes based control. But it requires some changes from the upper level rafthttp, which now has a message based buffer (N number of messages).